Ports/rvvm/package.sh

```diff
@@ -1,18 +1,19 @@
 #!/usr/bin/env -S bash ../.port_include.sh
+# Maintainer: LekKit <github.com/LekKit>
+
```

Comment:
We usually don't do `Maintainer` tags like this. If someone wants to maintain the port, they can do so without this comment, and all others are also free to create PRs that touch this file.